### ISSUE TITLE
find a way to handle all u_i function

### ISSUE THREAD

**Issue by 0Lucifer0** [OpenNos/OpenNos#162]
we need to find a way to handle all u_i function

**Comment by 0Lucifer0** [comment]
what about the best way to handle all u_i function?
when we click on item it send packet u_i
 (use item) so what's the best way to handle all item methods?
we can create an use_item static class with all methods called by the name of the item and if not found just show
 item useitem method Handler doesn't exist
 Something like this?

**Comment by kanam99** [comment]
blockquote {padding-left: 1ex; margin: 0px 0px 0px 0.8ex; border-left: #cccccc 1px solid;} p {margin: 0px;padding: 0px;} 
I just want to say that i realy appreciate your work on that project. Specialy for this project (and my plan for private NT server) I started learning C# (I was programming in C++ before). I will help you as much as I can as soon as I could. Unfortunately, I don't have expirence in team programming or finding bugs, altought I made few Metin2 private servers back in my primary-school time. Keep up your great work! Best wishes.
